Game/Pregnancy/MenstrualCycle.gd

```diff
@@ -134,11 +157,16 @@ func ovulate():
 	for orifice in OrificeType.getAll():
 		if(!hasWombIn(orifice)):
 			continue
-			
+		
+		var ch = getCharacter()
 		var amountOfEggs = RNG.pickWeightedPairs(possibleEggAmounts)
-		if(amountOfEggs == null):
-			amountOfEggs = 1
-		#print("AMOUNT OF EGGS: "+str(amountOfEggs))
+		amountOfEggs = max(ch.getMinEggsAmount(), int(round(amountOfEggs * ch.getEggsBonusMod())))
+		if(GM.pc.hasPerk(Perk.FertilityBetterOvulation) && amountOfEggs < 10):
+			amountOfEggs += RNG.randi_range(0, 4) #otherwise species with low base eggs like humans, won't get much bonus
```

Comment:
I would just use ceil() instead of round() for the amountOfEggs variable instead of hardcoding it like that